src/bundle/Resources/config/ezplatform_default_settings.yml

```diff
@@ -6,6 +6,7 @@ parameters:
     # Pagination limits
     ezsettings.default.search_pagination_limit: 10
     ezsettings.default.trash_pagination_limit: 10
+    ezsettings.default.section_pagination_limit: 10
```

Comment:
I'm not convinced with having different limit setting for every page? Why do we need to complicate it so much?

More important thing: you are defining parameters here but you haven't provided config parser for them which means it isn't possible to override those with config.yml. Please reference what I did with content ids.

Reply:
It is more flexibility doing this way, but config should be validated.